packages/local-signing-manager/src/types/index.ts

```diff
@@ -4,12 +4,15 @@
 export type PrivateKey =
   | {
       uri: string;
+      derivationPath?: string;
```

Comment:
If we do want derivationPath for uri, then we can do a union like

`export type PrivateKey =  ({ uri } | { mnemonic } | { seed }) & { derivationPath?: string}`

Same thing but a bit more succient + makes it easier to add TS Doc comment so people can see what it does in auto complete